edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.E2E.Test/EdgeHubFixtureCollection.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.E2E.Test
+{
+    using System.Collections.Generic;
+
+    public static class EdgeHubFixtureCollection
```

Comment:
Where was the existing behavior being a problem?

Reply:
Just responded on the other comment. The main reason I needed all this code was the backup and restore PR which required creating the EdgeHub app again in the same process.

Reply:
The changes in this specific PR were trying to make things simpler and solve another problem with test sequence failure.